src/components/Header-Termsofuse/HeaderTermsofuse.jsx

```diff
@@ -1,40 +1,41 @@
 import React from 'react';
-import termsofuseLogo from '../../images/termsofuse-logo.png';
-import tan from '../../images/tan.png';
+import termsofuseLogo from '../../images/logo-for-gallery.png';
 import favorite from '../../images/input-icon-first.svg';
 import arrow from '../../images/input-second-icon.svg';
+import tan from '../../images/tann.png';
 
 const HeaderTermsofuse = () => {
```

Comment:
When I comment out this component from `/termsofuse`, fotter is laso not shown. Looks like you have problem in design, please fix this